app/assets/stylesheets/application.scss

```diff
@@ -160,7 +160,7 @@ body {
       line-height: 6rem;
       font-weight: 900;
 
-      &.diy-header {
+      &.landing-page-header {
```

Comment:
just a little refactor here since we're using these styles now on 3 very similarly designed landing pages for DIY, Full Service, and EIP

Reply:
Happy to see this!